test/module/irohad/network/block_loader_test.cpp

```diff
@@ -82,6 +82,15 @@ class BlockLoaderTest : public testing::Test {
         .createdTime(iroha::time::now());
   }
 
+  rxcpp::observable<std::shared_ptr<shared_model::interface::Block>>
```

Comment:
`using wBlock` can be defined somewhere in test if it is not available, and `std::shared_ptr<shared_model::interface::Block>` can be replaced with `wBlock`